engine/src/main/java/org/entando/entando/aps/system/services/entity/AbstractEntityTypeService.java

```diff
@@ -100,15 +75,25 @@ protected EntityTypeShortDto toDto(IApsEntity src) {
         };
     }
 
-    protected PagedMetadata<EntityTypeShortDto> getShortEntityTypes(String entityManagerCode, RestListRequest requestList) {
+    protected PagedMetadata<EntityTypeShortDto> getShortEntityTypes(String entityManagerCode,
+                                                                    RestListRequest requestList) {
+
         IEntityManager entityManager = this.extractEntityManager(entityManagerCode);
-        List<IApsEntity> entityTypes = new ArrayList<>(entityManager.getEntityPrototypes().values());
+
         Map<String, String> fieldMapping = this.getEntityTypeFieldNameMapping();
-        entityTypes.stream().filter(i -> this.filterObjects(i, requestList.getFilters(), fieldMapping));
```

Comment:
This filter here doesn't work because the streams are lazy. It would only work if we added a collect and retrieved the result in a new variable.